stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -193,6 +193,24 @@ internal class StripePaymentController internal constructor(
         )
     }
 
+    override suspend fun confirmWechatPay(
+        confirmPaymentIntentParams: ConfirmPaymentIntentParams,
+        requestOptions: ApiRequest.Options
+    ): WechatPayNextAction {
+        confirmPaymentIntent(
+            confirmPaymentIntentParams,
+            requestOptions
+        ).let { paymentIntent ->
+            require(paymentIntent.nextActionData is StripeIntent.NextActionData.WechatPayRedirect) {
+                "Unable to confirm Payment Intent with WechatPay SDK"
```

Comment:
Let's make sure we capitalize the C in Chat everywhere

Reply:
hmm intentionally made it lowercased because the API treats it as a single word, should we keep it camel cased in side SDK?

e.g
```
"next_action": {
  "type": "wechat_pay_redirect_to_android_app",
  "wechat_pay_redirect_to_android_app": {
    "app_id": "wx65997d6307c3827d",
    "nonce_str": "some_random_string",
    "package": "Sign=WXPay",
    "partner_id": "wx65997d6307c3827d",
    "prepay_id": "test_transaction",
    "sign": "8B26124BABC816D7140034DDDC7D3B2F1036CCB2D910E52592687F6A44790D5E",
    "timestamp": "1619638941"
  }
},
```

Reply:
Feels like camel case is better because that's the actual name of the product. Maybe check with iOS?

Reply:
confirmed with iOS, they will go with `WechatPay`